cypress-tests/cypress/e2e/PaymentUtils/Commons.js

```diff
@@ -959,6 +983,134 @@ export const connectorDetails = {
         },
       },
     },
+    InvalidCurrency: {
+      Request: {
+        currency: "United",
+        payment_method: "card",
+        payment_method_type: "debit",
+        setup_future_usage: "on_session",
+        payment_method_data: {
+          card: {
+            card_number: "4242424242424242",
+            card_exp_month: "01",
+            card_exp_year: "2023",
+            card_holder_name: "joseph Doe",
+            card_cvc: "123456",
```

Comment:
Why can't we have one variation test case at a time , i can see you are using invalid currency and invalid cvv in one test case , can you clarify this ?

Reply:
The thing is, errors are thrown based on the priority or the way code is written.  
CVC is checked before expiry year is checked. So, in this case, it does not matter if we use an already expired year.  

Hope that clears things out.